src/components/views/settings/devices/FilteredDeviceList.tsx

```diff
@@ -185,9 +200,15 @@ export const FilteredDeviceList =
         saveDeviceName,
         onSignOutDevices,
         onRequestDeviceVerification,
+        setPusherEnabled,
+        supportsMSC3881,
     }: Props, ref: ForwardedRef<HTMLDivElement>) => {
         const sortedDevices = getFilteredSortedDevices(devices, filter);
 
+        function getPusherForDevice(device: DeviceWithVerification): IPusher | null {
+            return pushers.find(pusher => pusher[PUSHER_DEVICE_ID.name] === device.device_id);
+        }
```

Comment:
```suggestion
        const getPusherForDevice = (device: DeviceWithVerification): IPusher | null => {
            return pushers.find(pusher => pusher[PUSHER_DEVICE_ID.name] === device.device_id);
        };
```

I'm paranoid about scope changing randomly on us

Reply:
I'm not sure I understand the rationale behind this change. 
This function, overall declared will be scoped to the function it's declared in. Using `const` will scope it to the block it's declared in...
It's the whole concept of [closures](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Closures) and sounds more like a stylistic preference